telco-hub/configuration/reference-crs/required/acm/observabilityMCO.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: observability.open-cluster-management.io/v1beta2
+kind: MultiClusterObservability
+metadata:
+  name: observability
+  annotations:
+    mco-disable-alerting: "true"
```

Comment:
Please add a comment/explanation that this setting is required when the desired spoke cluster configuration includes tuning of the openshift monitoring components. This setting prevents ACM from overwriting the custom tuning but requires that the custom tuning include forwarding to ACM alert manager.